src/Lime.Transport.SignalR/ServerSignalRTransport.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Channels;
+using System.Threading.Tasks;
+using Lime.Protocol;
+using Lime.Protocol.Network;
+using Lime.Protocol.Serialization;
+using Microsoft.AspNetCore.Http.Connections;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.AspNetCore.SignalR;
+using Microsoft.AspNetCore.SignalR.Client;
+
+namespace Lime.Transport.SignalR
+{
+    /// <summary>
+    /// Provides methods for the server to communicate with client nodes using SignalR as the underlying transport mechanism.
+    /// </summary>
+    /// <inheritdoc />
+    internal sealed class ServerSignalRTransport : SignalRTransport
+    {
+        private const string FROM_SERVER_METHOD = "FromServer";
+        private const string CLOSE_METHOD = "Close";
+        private readonly IHubContext<EnvelopeHub> _hubContext;
+        private readonly string _connectionId;
+        private readonly HubOptions _hubOptions;
+        private readonly HttpConnectionDispatcherOptions _httpConnectionDispatcherOptions;
+
+        internal ServerSignalRTransport(
+            IHubContext<EnvelopeHub> hubContext,
+            string connectionId,
+            Channel<string> envelopeChannel,
+            IEnvelopeSerializer envelopeSerializer,
+            HubOptions hubOptions,
+            HttpConnectionDispatcherOptions httpConnectionDispatcherOptions,
+
+            ITraceWriter traceWriter = null) : base(envelopeChannel,
+                                                    envelopeSerializer,
+                                                    traceWriter)
```

Comment:
Fix indentation to keep the consistency
```suggestion
            ITraceWriter traceWriter = null) 
            : base(
                envelopeChannel,
                envelopeSerializer,
                traceWriter)
```